Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3
```

Comment:
This needs to use a most specific python version

Reply:
Bot is currently stable between versions 3.5.3 - 3.7.4 that I've checked, Be aware 3.6.0 has a windows flaw in it and will crash/error out systems using ws and gateways.

Reply:
Sounds good! I'll also look into what version is the most specific version. For NodeJS it's best to use the trimmed-down version of FROM:(Node) in the Dockerfile, I'll try to do the same here.

Reply:
Id probably say the best version of python to go with is 3.7.x as has some cleaner functions and is a lil more optimized.
Would be possible if you *really* wanted performance to use my 3.8 PyPy compiler but does require CPP build tools to compile everything for discord.py requirements